js/private/test/image/structure/digests.sum

```diff
@@ -1,2 +1,2 @@
-099acc2677cf8fbcd5c49d79169a21a8f541234d312e8e71b260af484093e061
+db4b47b2410911b723bed458f4be3ac6eae5859c232e924f7401856224233196
```

Comment:
This one again.. it also snuck into one of Derek's PRs.

Reply:
Can you undo the change to this file before landing?

Reply:
The build was red without this. What's the root cause of it?

I'm out atm but will try removing it in a bit

Reply:
I'm not sure. It is related to a change @thesayyn made recently. We'll have to sort it out as it is affecting our PRs.

Reply:
Looks like Derek fixed it in https://github.com/aspect-build/rules_js/pull/1012 so you may just be able to rebase.